unity3d/Assets/Plugins/iOS/swrve_sdk.mm

```diff
@@ -1,5 +1,8 @@
 #import <CoreTelephony/CTTelephonyNetworkInfo.h>
 #import <CoreTelephony/CTCarrier.h>
+#if !defined(SWRVE_NO_IDFA)
```

Comment:
@seaders @Sergio-Mira : the reason for #if !defined vs #ifndef is that it can be very hard to see the difference between #ifdef and #ifndef - it was something that we took with us when we moved from the previous company and writing lots of C / C++ code.